### DISCUSSION TITLE
Missing 'Manage Columns' section

### DISCUSSION THREAD

**Discussion by ellanrk** [microsoft-search/pnp-modern-search#3561] [Q&A] [answered]
Starting today morning, I don't see manage columns section.

![image](https://github.com/microsoft-search/pnp-modern-search/assets/159083804/dc76f6d6-4553-4bc3-8298-bc66fc8cae11)

**Comment by kasperbolarsen** [comment, marked as answer]
Have you selected the Custom Layout? As I recall it "Manage Columns" is only available in the Datails List Layout.

**Reply by ellanrk** [reply]
Thanks! Yes, I was in custom layout. I was able to see manage columns as soon as I switched back to details list layout.
